dbt-snowflake/src/dbt/include/snowflake/macros/relations/dynamic_table/replace/iceberg_managed.sql

```diff
@@ -0,0 +1,32 @@
+{% macro snowflake__replace_dynamic_table_iceberg_managed_sql(dynamic_table, relation, sql) -%}
+{#-
+    Produce DDL that replaces a dynamic iceberg table with a new dynamic iceberg table
+
+    This follows the syntax outlined here:
+    https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table#create-dynamic-iceberg-table
+
+    Args:
+    - dynamic_table: SnowflakeDynamicTableConfig - contains all of the configuration for the dynamic table
+    - relation: Union[SnowflakeRelation, str]
+        - SnowflakeRelation - required for relation.render()
+        - str - is already the rendered relation name
+    - sql: str - the code defining the model
+    Returns:
+        A valid DDL statement which will result in a new dynamic iceberg table.
+-#}
+
+{%- set catalog_relation = adapter.build_catalog_relation(config.model) -%}
+
+create or replace dynamic iceberg table {{ relation }}
+    target_lag = '{{ dynamic_table.target_lag }}'
+    warehouse = {{ dynamic_table.snowflake_warehouse }}
```

Comment:
no quotes?

Reply:
That's how it was before, this is just moved.